src/transformers/__init__.py

```diff
@@ -1921,6 +1927,15 @@
             "IBertPreTrainedModel",
         ]
     )
+    _import_structure["models.idefics"].extend(
+        [
+            "IDEFICS_PRETRAINED_MODEL_ARCHIVE_LIST",
+            "IdeficsForCausalLM",
+            "IdeficsGatedCrossAttentionLayer",
```

Comment:
```suggestion
```
We don't put layer objects in the main init anymore. It can be in the init of `models.idefics` however.